Adjust the drop-position of the air strike in D2k

abc013 opened the pull request:
Before change

![d2kbug](https://cloud.githubusercontent.com/assets/21260178/23139406/a9abe6d2-f7ac-11e6-9544-7036ed772072.gif)
---
After change

![d2kbugfix](https://cloud.githubusercontent.com/assets/21260178/23139524/17a65168-f7ad-11e6-9fae-7dbb078b6e3d.gif)

abc013 committed:
Adjust the drop-position of the air strike in D2k

Phrohdoh commented on mods/d2k/weapons/other.yaml:
```diff
@@ -67,7 +67,7 @@ OrniBomb:
 	Burst: 5
 	BurstDelay: 3
-	Range: 3c0
+	Range: 1c500
```
`1c512` is half of `3c0`.

abc013 commented:
Updated.

reaperrr committed:
Tweak D2k aircraft altitudes

The Frigate and Ornithopter were flying too low compared to the original.

reaperrr committed:
Tweak Ornithopter bombs

In the original,
- they were slower
- didn't accelerate
- had a shadow
- were dropped at lower rate
- played EXPLSML4.WAV

compared to bleed.

reaperrr commented:
This can be improved further, for example ornithopters currently fly too low and the bombs fall too fast, lack shadow and play the wrong sound on impact.

I've filed a PR to your branch that fixes these and slightly increases the distance between them.

In the future we should further increase the delay between them, but we need linear formation support and multi-flybys first, othwise they deal too little damage to the target position.

GraionDilach commented:
I have started working on multi-flybys but my method involves dropping AttackBomber out and I haven't looked at the Lua requirements to see what upstream requires for an 1:1 switch to my setup (I also have some bugs with guarding anyways).

abc013 commented:
@reaperrr looks good, merged it.

One question: in the original game, do the explosions of the bombs create craters?

abc013 commented:
![d2kbugfix gif](https://cloud.githubusercontent.com/assets/21260178/23167442/9c93c1fc-f844-11e6-9ffb-826e045cd583.gif)

reaperrr commented:
> One question: in the original game, do the explosions of the bombs create craters?

Yes, they did, I just forgot to check for that.

reaperrr commented:
As it turns out, the craters are not appearing because the OrniBomb weapon (other.yaml) defines a bogus 'Crater' smudge type. The correct setting would be `SmudgeType: SandCrater, RockCrater`

Additionally, I'd suggest to add `SquadOffset: -1536, 1024, 0` to the AirstrikePower, to make the attacked area exactly 3x3 cells (in the above gif, the outer ornis would drop their bombs in the middle of the turret cells instead of on the outer border).

👍 after that.

abc013 commented:
![d2kbugfix3](https://cloud.githubusercontent.com/assets/21260178/23334479/fadd7100-fb9f-11e6-91f4-f05bf7da33b6.gif)

Are you sure you dont mean 4x3 instead of 3x3?

reaperrr commented:
> Are you sure you dont mean 4x3 instead of 3x3?

You're right, of course, I was only thinking about the "width".

reaperrr commented:
@GraionDilach Just to be sure, do you consider your +1 still valid after the changes?

GraionDilach reviewed (approved):
I didn't. Now I do.

reaperrr commented:
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/d703ee5d46968382725567db546c4449ecdeab3a...190fae97a20f105f49e27d2ea0af3e5c3731d954)
